Jenkinsfile

```diff
@@ -45,6 +45,7 @@ pipeline {
         stage('Create build environment') {
             options {
                 timeout(time: 60, unit: "MINUTES")
+                timestamps ()
```

Comment:
Do we only want that in this stage or in all of them?

The commit message should have more information about why this is useful, and "try" looks like a left-over from experimenting.